packages/protocol/contracts/identity/Attestations.sol

```diff
@@ -680,6 +698,17 @@ contract Attestations is
       msg.sender == identifiers[identifier].accounts[index],
       "Index does not match msg.sender"
     );
+    require(
+      transferAllowed[replacementAddress][msg.sender] == true,
```

Comment:
@bohan-zhang is the implication of having the `transferAllowed` that the `msg.sender` could now be the Onboarding contract?  In which case, we should change all these `msg.sender` in this `transfer` method to an `address originalAddress` parameter

Reply:
cc @bowd ?

Reply:
lol whoop